### ISSUE TITLE
Please can you help us with creating a training dataset without reverberations?

### ISSUE THREAD

**Issue by GSC-30212** [microsoft/DNS-Challenge#38]
Please can you help us with creating a training dataset without reverberations?
Because looking at the code of 'noisyspeech_synthesizer_singleprocess.py' and configuration 'noisyspeech_synthesizer.cfg' we cannot find a setting for the same.

The parameter 'rir_choice' always needs to be specified as : -
 for only real rir, 2 for only synthetic rir, 3 (default) use both real and synthetic
but no value for 'no reverberation'. As specified in the paper the 'DNS test set no reverb'.

![pesq](https://user-images.githubusercontent.com/35251867/104449517-b7c4a180-55c4-11eb-9a7a-ddbe9629a5be.png)

**Comment by Rodolfo-S** [comment]
I think you can just comment out [Line 188](https://github.com/microsoft/DNS-Challenge/blob/959a6f4371f9173d7e0add886ff7d32107994566/noisyspeech_synthesizer_singleprocess.py#L188) in `noisyspeech_synthesizer_singleprocess.py`. That's the line that overwrites the clean signal with clean convolved with the RIR.
